GoMeet/src/main/webapp/js/login-status.js

```diff
@@ -0,0 +1,6 @@
+class LoginStatus {
+  static async doGet(servlet) {
```

Comment:
Optional: Is this just going to be used to make requests to '/user-status'? In that case, could consider making '/user-status' a static variable in this class (similar to the Meeting-time-DAO). I've done something similar for meeting-location-dao. I think this would make the project for uniform overall.

Reply:
Also, in future, I think this class would be a good place to add any checks i.e. unexpected responses from the server.

Reply:
Noted :)